source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRSeparator.mm

```diff
@@ -0,0 +1,170 @@
+                separator.lineWidth = separatorHstCnfig->lineThickness;
+                separator.backgroundColor = UIColor.clearColor;
+                [superView addArrangedSubview:separator];
+                
+                NSLayoutConstraint* constraint = nil;
+                separator.axis = superView.axis;
+                if(UILayoutConstraintAxisVertical == superView.axis)
+                {
+                    constraint = [NSLayoutConstraint constraintWithItem: separator
+                                                              attribute: NSLayoutAttributeWidth
+                                                              relatedBy: NSLayoutRelationEqual
+                                                                 toItem: superView
+                                                              attribute: NSLayoutAttributeWidth
+                                                             multiplier: 1
+                                                               constant: 0];
+                    [separator setContentHuggingPriority:UILayoutPriorityDefaultLow forAxis:UILayoutConstraintAxisHorizontal];
+                    [separator setContentCompressionResistancePriority:UILayoutPriorityDefaultLow forAxis:UILayoutConstraintAxisHorizontal];
+                    [separator setContentHuggingPriority:UILayoutPriorityDefaultHigh forAxis:UILayoutConstraintAxisVertical];
+                    [separator setContentCompressionResistancePriority:UILayoutPriorityDefaultHigh forAxis:UILayoutConstraintAxisVertical];
+                }
+                else
+                {
+                    constraint = [NSLayoutConstraint constraintWithItem: separator
+                                                              attribute: NSLayoutAttributeHeight
+                                                              relatedBy: NSLayoutRelationEqual
+                                                                 toItem: superView
+                                                              attribute: NSLayoutAttributeHeight
+                                                             multiplier: 1
+                                                               constant: 0];
+                    [separator setContentHuggingPriority:UILayoutPriorityDefaultHigh forAxis:UILayoutConstraintAxisHorizontal];
+                    [separator setContentCompressionResistancePriority:UILayoutPriorityDefaultHigh forAxis:UILayoutConstraintAxisHorizontal];
+                    [separator setContentHuggingPriority:UILayoutPriorityDefaultLow forAxis:UILayoutConstraintAxisVertical];
+                    [separator setContentCompressionResistancePriority:UILayoutPriorityDefaultLow forAxis:UILayoutConstraintAxisVertical];
+                }
+                
+                if(constraint) [superView addConstraint:constraint];
+            }
+            catch(...) {;};
```

Comment:
>catch(...) {;}; [](start = 12, length = 15)

What exceptions are you intending to catch here? Bad input to stoul? Silently dropping the exception is probably not the best thing.

If the string being parsed is expected to be in the correct format, then don't catch the exception at all; this kind of invariant failure warrants terminating the process. If the string being parsed could reasonably be incorrect -- if it's read from external input, say -- then that kind of bad input should be handled in some way. Logged, at the very least.